### DISCUSSION TITLE
How can I override default terms?

### DISCUSSION THREAD

**Discussion by orenard** [sapegin/textlint-rule-terminology#69] [General]
Hi, the default terms contain an entry that I would like to override. How can this be done? The words in question "code base", which Textlint does not like and would like to shorten to "codebase". I have tried to define my own `.textlintrc.json` containing this:

`{
  "rules": {
    "terminology": {
      "defaultTerms": false,
      "exclude": ["codebase"]
    }
  }
}
` 

But it does not have any effect.

Please note that I am using Textlint through Super Linter.

Can you please let me know how this can be done?

Many thanks

**Comment by rfay** [comment]
@sapegin could you suggest how to override?

In https://github.com/sapegin/textlint-rule-terminology/commit/787da86db165e0cf46d8e3680809a091238ce482#diff-a20da60dfec71fb7418916186c1249fe4746058a7f50348a43cd17f6157caca0R85

you added `  ["V[ -]?S[ -]?Code", "Visual Studio Code"],`

We use "VS Code" in our DDEV docs, and would like to continue doing so, as that's the normal reference people use, and it works best in search. 

But unfortunately, once you've added that rule, it seems there's no way to override it. The only thing I could do was to completely turn off `defaultTerms`:
```
    "terminology": {
      "defaultTerms": false,
```

I tried all kinds of other approaches.

**Comment by sapegin** [comment]
This works, and documented:

```js
{
  rules: {
    terminology: {
      exclude: ['V[ -]?S[ -]?Code']
    },
}
```

I have updated the docs too to make it a bit more clear.

**Reply by rfay** [reply]
Thanks so much @sapegin - The key is that you have to look find exactly how the rule is specified in https://github.com/sapegin/textlint-rule-terminology/blob/master/terms.jsonc - so you have to look at terms.json and use exactly the format shown.  I tried many, many things before understanding that the match is on the format in the terms.jsonc, not on the matched text. So I was trying to exclude "VS Code" several different ways, but that's not what's specified in terms.jsonc, so it could never work. In this specific case, as you point out, it had to be done with the format in terms.jsonc, `'V[ -]?S[ -]?Code'`
